src/openrct2/world/Sprite.h

```diff
@@ -74,14 +74,20 @@ struct Duck : SpriteGeneric
 
 struct MoneyEffect : SpriteBase
 {
-    uint16_t move_delay;
-    uint8_t num_movements;
-    uint8_t vertical;
-    money32 value;
-    int16_t offset_x;
-    uint16_t wiggle;
-
-    static void CreateAt(money32 value, int32_t x, int32_t y, int32_t z, bool vertical);
+    enum class TransactionType : bool
+    {
+        CREDIT,
+        DEBIT,
+    };
+
+    uint16_t MoveDelay;
+    uint8_t NumMovements;
+    TransactionType CashFlow;
```

Comment:
I am not entirely convinced this is indeed credit/debit. Would have to check the calls. I am not sure about the name of this field either.